### PR TITLE
Refactor/devevang 100 python rewrite

### PR DESCRIPTION
**Pull request by marlowetutor**
Pull Request for DEVEVANG 100 fix
1. Recoded the AWS Lambda implementation to use AWS SAM instead
2. Modified the index.md to reflect all the changes

### CODE REVIEW

File: blueprint/aws-sam-email-classifier/emailclassifier/__init__.py
Comment (kavinbalagen) [inline blueprint/aws-sam-email-classifier/emailclassifier/__init__.py:1]: this maybe redundant file. check if you can still deploy the lambda without this.

File: blueprint/aws-sam-email-classifier/template.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        UsagePlan:
+          CreateUsagePlan: PER_API
+          UsagePlanName: GatewayAuthorization
+
+  EmailClassifierFunction:
+    Type: AWS::Serverless::Function # More info about Function Resource: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#awsserverlessfunction
+    Properties:
+      CodeUri: emailclassifier/
+      Handler: app.lambda_handler
+      Runtime: python3.9
```
Comment (kavinbalagen) [inline blueprint/aws-sam-email-classifier/template.yaml:29]: can we update the runtime to python3.12 if possible?

File: blueprint/aws-sam-email-classifier/events/event.json
Comment (kavinbalagen) [inline blueprint/aws-sam-email-classifier/events/event.json:1]: are we using this event file anywhere?

**Review by kavinbalagen — APPROVED** [review]
I have added a few comments, overall code looks good to me.

File: blueprint/aws-sam-email-classifier/emailclassifier/__init__.py
Comment (marlowetutor) [reply blueprint/aws-sam-email-classifier/emailclassifier/__init__.py:1]: The deployment was successful even when the file was deleted, File will be omitted on next commit

File: blueprint/aws-sam-email-classifier/template.yaml (hunk 1 above)
Comment (marlowetutor) [reply blueprint/aws-sam-email-classifier/template.yaml:29]: Python 3.9 is required when using AWS SAM. I have tried to use later versions, but it seems to be incompatible.

Comment (carnellj-genesys) [reply blueprint/aws-sam-email-classifier/template.yaml:29]: Agree with Marlowe

**Review by carnellj-genesys — CHANGES_REQUESTED** [review]

File: blueprint/aws-sam-email-classifier/events/event.json
Comment (marlowetutor) [reply blueprint/aws-sam-email-classifier/events/event.json:1]: It is used by the lambda function

**Review by carnellj-genesys — APPROVED** [review]

**Pull request merged**